src/containers/OwnerView.js

```diff
@@ -60,41 +68,53 @@ function OwnerView({ owner }) {
     .filter(character => character.owner === owner)
     .map(character => (
       <CharacterSummary
-        key={ character.id }
-        character={ character }
-        highlight={ party.indexOf(character.id) !== -1 }
-        add={ () => toggleCharacter(character.id) }
-        linkTo={ `/character?id=${ character.id }` }
+        key={character.id}
+        character={character}
+        highlight={party.indexOf(character.id) !== -1}
+        add={() => toggleCharacter(character.id)}
+        linkTo={`/character?id=${character.id}`}
       />
     ));
 
   const otherCharacters = characters
     .filter(character => character.owner !== owner)
     .map(character => (
       <CharacterSummary
-        key={ character.id }
-        character={ character }
-        highlight={ party.indexOf(character.id) !== -1 }
-        add={ () => toggleCharacter(character.id) }
-        linkTo={ `/character?id=${ character.id }` }
+        key={character.id}
+        character={character}
+        highlight={party.indexOf(character.id) !== -1}
+        add={() => toggleCharacter(character.id)}
+        linkTo={`/character?id=${character.id}`}
       />
     ));
 
+  const ownerParties = Object.keys(parties).length > 0 ? Object.keys(parties).map(key => (
```

Comment:
if the length is 0 wouldn't it just return an empty array which would display the same as null? just wondering if the check is necessary but i may be wrong.

Reply:
This was just to make sure that the Party Bar that holds the chips doesn't render when there are no parties.

Reply:
so up here its just the map, so it will be an empty or populated array and down below you check `!!ownerParties.length`

Reply:
ahhh sweet i don't think ive seen that short hand

Reply:
Honestly you don't even need the double unary (`!!`) thats just to make sure it converts to a boolean, but the raw falsey should also technically work (0 being falsey and every other int being truthy) but sometimes react will actually display the `0` if you do that which is annoying